src/che-openshift-authentication.ts

```diff
@@ -84,7 +87,11 @@ export async function start(context: theia.PluginContext) {
             theia.window.showErrorMessage(errorMessage + e);
             return;
         }
-        const osCommand = spawn('oc', ['login', server, '--certificate-authority=/var/run/secrets/kubernetes.io/serviceaccount/ca.crt', '--token', token]);
+        const args = ['login', server, '--token', token];
+        if (!isHostedChe) {
+            args.push('--certificate-authority=/var/run/secrets/kubernetes.io/serviceaccount/ca.crt');
```

Comment:
Could you extract the path to a constant? Just for paying attention on it, if the path is not changed.

Reply:
I prefer to keep it inline, this path is used only in a single place.